[Docs] Add details on storing alerts and alert reset (#7470)
mlrun/mlrun: docs/concepts/alerts.md

Alerts are a mechanism for informing you about possible problem situations. 

**In this section**
- [System configuration](#system-configuration)
- [SDK](#sdk)
- [Predefined events](#predefined-events-eventkind)
- [Creating an alert](#creating-an-alert)

> - [Modifying alerts](#modifying-alerts)

Reviewer: ```suggestion
- [Modifying an alert](#modifying-an-alert)
```

> - [Modifying alerts](#modifying alertsß)

Reviewer 2: ```suggestion
- [Modifying alerts](#modifying-alerts)
```

project.store_alert_config(alert_data)
```

## Modifying alerts

> When you run `store_alert_config`:

Reviewer: ```suggestion
When you run `store_alert_config` on an existing alert:
```

> - The alert is reset if you modify a field that relates to the conditions that trigger the alert. These fields are:

Reviewer: ```suggestion
- The alert is reset if you modify a field that affects the conditions that trigger the alert. These fields are:
```
